include/envoy/event/file_event.h

```diff
@@ -7,6 +7,7 @@ namespace Event {
 struct FileReadyType {
   static const uint32_t Read = 0x1;
   static const uint32_t Write = 0x2;
+  static const uint32_t Closed = 0x4;
```

Comment:
Suggest making the documentation below a little clearer when distinguishing between activate/setEnabled. Right now, both seem like they might do something similar (enable/activate). I'd make clear that activate is "injecting the event" and setEnabled is effectively "listening for the event".